engines/bastion_katello/app/assets/javascripts/bastion_katello/debs/details/views/deb-info.html

```diff
@@ -9,22 +9,31 @@ <h4 translate>Basic Information</h4>
       <dd>
         <i class="fa fa-spinner fa-spin" ng-show="installedDebCount === undefined"></i>
         <span ng-show="installedDebCount !== undefined">
-          <a href="{{ '/content_hosts?search=' + createSearchString('installed_deb') }}" translate>
+          <a ng-if="newHostDetailsUI" href="{{ '/hosts?search=' + createSearchString('installed_deb') }}" translate>
```

Comment:
I think newHostDetailsUI isn't being sent  over somehow, since debs are still linking to the content hosts page.

Reply:
Need to add newHostDetailsUI to the `DebController.$inject` at the bottom of deb.controller.js.